spaces/domains.json

```
@@ -496,6 +496,8 @@
   "gov.illuvium.io": "ilvgov.eth",
   "vote.yzydao.finance": "yzydao.eth",
   "gov.premia.finance": "premia.eth",
+  "gov.premia.blue": "premia.eth",
```

You don't need to point 2 sub-domains to the same ENS.
```suggestion
  "gov.premia.blue": "premia.eth",
```